pkgs/unified_analytics/lib/src/event.dart

```diff
@@ -399,6 +399,7 @@ final class Event {
     int? rootSetCount,
     int? rowCount,
     int? inspectorTreeControllerId,
+    bool? isV2Inspector,
```

Comment:
after going through the churn of needing to update unified_analytics each time we want to track something new, I propose we add a map to the constructor `Map<String, Object?> additionalMetrics = const {}` and then add it to the `eventData` map like:

`...additionalMetrics..removeWhere((key, value) => value == null)`

and then we can deprecate all of the screen metrics here (from `uiDurationMicros` down to the bottom). These values are likely to change over time as we iterate on the different DevTools pages, and this would prevent us from having to change the unified_analytics package every time we need to add / remove a param. 

WDYT?